nekoyume/battle/simul.py

```diff
@@ -65,17 +80,18 @@ def simulate(self):
 class DummyBattle(Simulator):
     def __init__(self, seed):
         super().__init__(seed, 'zone_0')
-        self.characters.append(Factory.create_player(
+        factory = Factory()
+        self.characters.append(factory.create_player(
             'dummy_swordman', 'swordman', 1, ['taunt', 'attack'],
             [Weapon('sword_1')]))
-        self.characters.append(Factory.create_player(
+        self.characters.append(factory.create_player(
             'dummy_mage', 'mage', 1, ['firewall', 'attack'], []))
-        self.characters.append(Factory.create_player(
+        self.characters.append(factory.create_player(
             'dummy_acolyte', 'acolyte', 1, ['heal', 'attack'], []))
-        self.characters.append(Factory.create_player(
-            'dummy_archer', 'archer', 1, ['attack'], []))
-        self.characters.append(Factory.create_monster('slime'))
-        self.characters.append(Factory.create_monster('slime'))
-        self.characters.append(Factory.create_monster('slime'))
-        self.characters.append(Factory.create_monster('slime'))
-        self.characters.append(Factory.create_monster('griffin'))
+        # self.characters.append(factory.create_player(
+        #    'dummy_archer', 'archer', 1, ['attack'], []))
+        self.characters.append(factory.create_monster('slime'))
+        self.characters.append(factory.create_monster('slime'))
+        # self.characters.append(factory.create_monster('slime'))
+        # self.characters.append(factory.create_monster('slime'))
```

Comment:
Lines commented out could be removed at this time; we can recover them anytime we want from the Git history.